commandCenter/cleanup.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+rm -vf ./region
+
+cd ../data/
+rm -vf evtstaminmax nevtstaray receivers sources traveltimeReceiverGathers traveltimeReceiverGathers_synthetic
+
+cd ./dataSelection/1_picks/
+rm -vf largeEarthquakes  obstime-rec-gather  obstime-src-gather  phaseDataInResearchRegion
+cd ../../
+
+cd ../figure/
+rm -vf dvelCut* tomo.grd
```

Comment:
What's vz1d? Remove?
```suggestion
rm -vf dvelCut* tomo.grd vz1d
```